### PR TITLE
PIP-2.md and PIP-3.md English Translation

### PR DESCRIPTION
**Pull request by xiyu1984**
PIP-2.md English Translation

### CODE REVIEW

File: PIPs/PIP-2.md [hunk 1]
```diff
@@ -1,42 +1,43 @@
...
-- 区块头时间戳不同，以太网为秒，PlatON为毫秒
 
-由以上差异可知，PlatON只需要极小的改动就可以实现对以太坊的兼容。
+- The address format is different. PlatON uses the address format of Bech32, while Ethereum uses the address format of Eip55
+- The Token units are different. PlatON's Token units are LAT/VON and Ethereum's are Ether/WEI
+- Some functions are not implemented in Platon or are implemented differently. Functions such as block.difficlty, miner. SetEtherbase, etc., the Ethereum contracts using these functions may not work on PlatON and may need to be adjusted accordingly, but these functions are rarely used.
+- The block header timestamp is different, Ethereum in seconds and PlatON in milliseconds
 
-## 实现说明
+From these differences, Platon can be made compatible with Ethereum with minimal changes.
```
Comment (benbaley) [inline PIPs/PIP-2.md]: Platon should be PlatON

**Review by benbaley — CHANGES_REQUESTED** [review]
please rename the filename to PIP-2-EN.md

**Comment by biganxin** [comment]
> please rename the filename to PIP-2-EN.md
This English translation is recommended without adding a new document
Just add the English translation to the original Chinese document

**Review by benbaley — APPROVED** [review]
LGTM

**Pull request merged**